### DISCUSSION TITLE
Using the Tina Data Layer "Stand Alone"

### DISCUSSION THREAD

**Discussion by logan-anderson** [tinacms/tinacms#4236] [Ideas]
Many of our customers love using our Data Layer because it enables them to handle Markdown, JSON, and MDX at scale with confidence. While the Data Layer is integral to our CMS user interface functionality, we are considering the possibility of providing customers with standalone access to it.

This would allow users to  [utilize the Data Layer](https://tina.io/blog/self-hosted-datalayer/) without the need to build the [CMS frontend](https://tina.io/docs/using-tina-editor/). Additionally, users would have the option to provide their own database and self-host the data layer.

What could this potential offering look like?

Here's a brief demo video illustrating how this concept of a standalone Datalayer could function: [Demo Video](https://youtu.be/5FYTVcFKvg8).

Before we dive into development, we highly value your input. What are your thoughts on this feature? Do you find it useful? Your feedback is crucial to us

**Comment by bauer01** [comment]
I like this idea very much!...I'm currently little bit struggling with implementing self-hosted solution with serverless and [SST](https://sst.dev/) ... have you tried it @logan-anderson?

**Reply by logan-anderson** [reply]
@bauer01 We have not tried SST are you hosting a next app with SST? 

We are in the process of making self hosted more first class so expect some updates and a much easier experience in the next 2-3 weeks.
